paasta_tools/api/client.py

```diff
@@ -21,9 +21,9 @@
 from typing import Any
 from urllib.parse import urlparse
 
-from bravado.client import SwaggerClient
-
 import paasta_tools.api
+from bravado.client import SwaggerClient
```

Comment:
100500% nit: I was under impression that we keep different kind of imports in separate sections, that's way `bravado.client` was in it's own section: because it isn't a part of platform like`typing` and `urllib`, and also not a part of current project like `paasta_tools`, but a third-party library.  If that's true, we probably better keep that separation here.

Reply:
Sure, I guess we kinda care but it's tricky to enforce in pre-commit!